test/e2e/nftables_test.go

```diff
@@ -69,15 +102,14 @@ var _ = Describe("Nftables", func() {
 		// Wait for all goroutines to finish
 		err = g.Wait()
 		Expect(err).ToNot(HaveOccurred())
-		nodeName := nodeList.Items[0].Name
 
-		By("Rebooting first node: " + nodeName + "and waiting for disconnect")
+		log.Printf("Waiting for %s after applying MachineConfig...\n", waitDuration)
```

Comment:
Why wait (sleep)? A more solid approach would be to confirm that MCP has started updating.

Reply:
removed i have the Eventually wait

Reply:
You missed the point. If you jump right to the Eventually, the MachineConfig Operator may have not started updating yet leading to the Eventually function exiting immediately but wrongly.

Reply:
LGTM :+1: